xep-0393.xml

```diff
@@ -255,6 +267,19 @@
     </di>
   </dl>
 </section1>
+<section1 topic='Discovering Support' anchor='disco'>
+  <p>
+    Clients that support Extensible IBR MUST advertise the fact by including a
```

Comment:
Extensible IBR? I suppose this was meant to say "Message Styling"?

Reply:
Oops, copy/pasta. Fixed. Thanks!